### ISSUE TITLE
Outdated Library can couse error during ILIAS 7 migrations

### ISSUE THREAD

**Issue by chfsx** [opencast-ilias/OpenCast#147]
**Describe the bug**
As described in https://mantis.ilias.de/view.php?id=35910, an outdated version of a Symfony library can prevent on migrate in ILIAS core. 

**To Reproduce**
Currently I wasn't able to reproduce this, but nevertheless I suggest to update the library

**Expected behavior**
New version of the library is deployed with the plugin

**Screenshots**
-

**Log**
```
Migration of File-Objects to Storage service...
[FAILED]
#0 ILIAS\Setup\CLI\MigrateCommand-›achieveObjective() called at I/var/www/ilias/src/Setup/CLI/MigrateCommand.php: 113]
#1 ILIAS\Setup\CLI\MigrateCommand-›runMigration()calledatl/var/www/ilias/src/Setup/CLI/MigrateCommand.php:64j
#2 ILIAS\Setup\CLI\MigrateCommand-›execute () called at [/var/www/ilias/libs/composer/vendor / symfony/ console / Command/ Command. php: 255]
#3 Symfony\Component \ Console \Command \Command-> run () called at [/var/www/ilias/libs/composer/vendor/symfony/console/Application.php:1021]
#4 Symfony\ Component \Console \Application-›doRunCommand ( called at [/var/www/ilias/libs/composer/vendor/symfony/console/Application.php:275]
#5 Symfony\Component \Console\Application-›doRun () called at [/var/www/ilias/libs/composer/vendor/symfony/console/Application.pho: 1491
#6 Symfony\ Component \Console \Application-›run () called at [/var/www/ilias/setup/cli.php:391
#7 require once l/var/www/ilias/setup/cli.pho) called at [/var/www/ilias/setup/setup.ohp: 421
[ERROR] Call to undefined method Symfony\ Component \Console \Helper \Helper: :width ()
```

**Environment (please complete the following information):**
 - ILIAS version:7.16
 - PHP version: unknown
 - Plugin version: unknown
 - Opencast version: unknown
 - Browser: unknown

**Additional context**
-

**Comment by chfsx** [comment]
We are no longer able to reproduce this, if anyone runs into the same problem, feel free to reopen the issue.

**Comment by fsesterh** [comment]
I can.... I am working on updating my DEV to ILIAS 8. During ILIAS 8 Setup I got the message that some migrations for ILIAS 7 had not been run, so I reset my VM to ILIAS 7 and ran php setup/setup.php migrate. Output:

 ! [NOTE] There are 3 to run:

 ilFileObjectMigrationAgent.ilFileObjectToStorageMigration: Migration of File-Objects to Storage service [remaining steps: 7]
 resourcestorage.ilStorageHandlerV1Migration: ilStorageHandlerV1Migration [remaining steps: 15]
 ilStudyProgrammeMigrationAgent.PRGUpdateCRSRefLPSettingMigration: Update LP Settings of Course References [remaining steps: 4]

 ! [NOTE] Run them by passing --run <migration_id>, e.g. --run
 !        ilStudyProgrammeMigrationAgent.PRGUpdateCRSRefLPSettingMigration

When I run ANY of the three steps they all crash with:

 [ERROR] Call to undefined method Symfony\Component\Console\Helper\Helper::width()

My Installation was updates from ILIAS 6 to ILIAS 7, the OpenCast Plugin was temporarily installed. I uninstalled it months ago. Seems like it stayed in part and causes trouble?
